scalatags/src/scalatags/VirtualDom.scala

```diff
@@ -107,14 +107,14 @@ trait VirtualDom[Output <: FragT, FragT]
 
     implicit class SeqFrag[A](xs: Seq[A])(implicit ev: A => Frag) extends Frag{
       Objects.requireNonNull(xs)
-      def applyTo(t: vdom.Builder[Output, FragT]): Unit = xs.foreach(_.applyTo(t))
+      def applyTo(t: vdom.Builder[Output, FragT]): Unit = xs.foreach(ev(_).applyTo(t))
```

Comment:
@jodersky is it reasonable to explicitly call the implicit, style-wise?

Reply:
I don't see any problem with it style-wise, but is it needed? Was there a change in Dotty that requires the explicit call here?

Reply:
Seems like implicit search works differently. Without the explicit call to `ev` dotty gives the following error:

```
[error] -- [E008] Not Found Error: /home/manojo/workspace/scalatags/scalatags/src/scalatags/VirtualDom.scala:110:71 
[error] 110 |      def applyTo(t: vdom.Builder[Output, FragT]): Unit = xs.foreach(_.applyTo(t))
[error]     |                                                                     ^^^^^^^^^
[error]     |value applyTo is not a member of A, but could be made available as an extension method.
[error]     |
[error]     |The following import might make progress towards fixing the problem:
[error]     |
[error]     |  import geny.Generator.from
```